app/database/itemStore.go

```diff
@@ -38,6 +38,30 @@ func (s *ItemStore) Insert(data *Item) error {
 	return s.db.insert(s.tableName(), data)
 }
 
+// HasManagerAccess returns whether the user has manager access to all the given item_id's
+// It is assumed that the `OwnerAccess` implies manager access
+func (s *ItemStore) HasManagerAccess(user *auth.User, itemID int64) (found bool, allowed bool, err error) {
+
+	var dbRes = []struct {
+		ItemID        int64 `sql:"column:idItem"`
+		ManagerAccess bool  `sql:"column:bManagerAccess"`
+		OwnerAccess   bool  `sql:"column:bOwnerAccess"`
+	}{}
+
+	db := s.GroupItems().MatchingUserAncestors(user).
+		Select("idItem, bManagerAccess, bOwnerAccess").
+		Where("groups_items.idItem = ?", itemID).
```

Comment:
```suggestion
		Where("idItem = ?", itemID).
```

Reply:
it depends a bit what's in "MatchingUserAncestors"  but apparently that's ok here